packages/atlas/src/MainLayout.tsx

```diff
@@ -6,6 +6,7 @@ import { StudioLoading } from '@/components/_loaders/StudioLoading'
 import { CookiePopover } from '@/components/_overlays/CookiePopover'
 import { atlasConfig } from '@/config'
 import { BASE_PATHS, absoluteRoutes } from '@/config/routes'
+import useAnalytics from '@/hooks/useSegmentAnalytics'
```

Comment:
It was missed before, but we should avoid default exports. Also, let's name the hook `useSegmentAnalytics`